dts/bindings/charger/ti,bq25180.yaml

```diff
@@ -30,7 +30,11 @@ properties:
       driver initialization.
 
   constant-charge-voltage-max-microvolt:
-    required: true
+    type: int
+    default: 4200000
+    description: |
+      The maximum voltage that the battery will be charged at, defaults to
+      4.2V.
```

Comment:
https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#rules-for-default-values :)

Reply:
fixed